interfaces/builtin/u2f_devices.go

```diff
@@ -62,6 +62,11 @@ var u2fDevices = []u2fDevice{
 		VendorIDPattern:  "096e|2ccf",
 		ProductIDPattern: "0880",
 	},
+	{
+		Name:             "HyperSecu HyperFIDO Pro",
+		VendorIDPattern:  "2ccf",
+		ProductIDPattern: "0854",
```

Comment:
I can't seem to find any reference to this being the official product ID for this device, I can only find references to the non-Pro version with product ID `0880`. Are you able to provide any sort of reference that can confirm that this is in fact the right product ID/device combination?

Reply:
In [the bug report](https://launchpad.net/bugs/1919268) I linked, in comment #1, there's a reference, although it's not official documentation. That product ID was later confirmed by the bug reporter in comment #2.